glean-core/ffi/src/lib.rs

```diff
@@ -409,6 +409,16 @@ pub extern "C" fn glean_is_dirty_flag_set() -> u8 {
     with_glean_value(|glean| glean.is_dirty_flag_set())
 }
 
+#[no_mangle]
+pub extern "C" fn handle_client_active() {
```

Comment:
This needs to be `glean_` prefixed:

```suggestion
pub extern "C" fn glean_handle_client_active() {
```

Reply:
which brings up the questions once again: does it need the `client` in there?

Reply:
`glean_handle_active` seems to imply glean becomes active. But we're really looking for the product/client activity. I'm open to better wording :)

Reply:
hm, why don't you want an active glean?!?!?!

Reply:
Too much sport :) Your call!

Reply:
After discussion in the channel we settle with the chosen name here.